### PR TITLE
Add test_clean_load

### PR DESCRIPTION
**Pull request by anikielx**
Signed-off-by: Adriana Nikelska <adrianax.nikielska@intel.com>

### CODE REVIEW

File: test/functional/tests/lazy_writes/test_clean_load.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        fio = Fio().create_command()\
+            .size(Size(1, Unit.GibiByte))\
+            .read_write(ReadWrite.randwrite)\
+            .io_engine(IoEngine.libaio)\
+            .block_size(Size(1, Unit.Blocks4096))
+        for i, core in enumerate(cores):
+            fio.add_job(f"core{i}").target(core.path)
+        fio.run()
+
+        if cache.get_dirty_blocks() <= Size.zero():
```
Comment (mmichal10) [inline test/functional/tests/lazy_writes/test_clean_load.py:57]: How about explicitly testing both cores for presence of dirty data?

**Comment by mmichal10** [comment]
ok to test

**Comment by mmichal10** [comment]
ok to test

**Comment by mmichal10** [comment]
I think you accidentally overwrited the original commit message :)

**Review by mmichal10 — APPROVED** [review]

**Review by katlapinka — APPROVED** [review]

**Pull request merged**